Automattic/wp-calypso
# Add email stats countries UI

#### Proposed Changes

* This PR adds the UI for the country view of email stats

#### Testing Instructions

* Visit `/stats/email/open/<site>/day/<postid>?flags=newsletter/stats&startDate=2022-12-28` (change `startDate` to a date where you have email opens)
* You should see the country map & list

![CleanShot 2023-01-03 at 11 28 11@2x](https://user-images.githubusercontent.com/528287/210339836-68ba3716-3ced-4b70-ac64-d70a13da983c.png)

* Ensure that the GeoChart component still works for site stats by visiting `/stats/day/<site>`

#### Pre-merge Checklist



- [X ] Have you checked for TypeScript, React or other console errors?
